credentials.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We should ignore this file and use env vars, since this is sensitive information, and should not be tracked.